pages/super-admin/product-listing/index.tsx

```diff
@@ -13,4 +14,4 @@ const ProductListing = () => {
     </div>
   );
 };
```

Comment:
Product listing is not the only admin route, please can you wrap the other admin routes with the HOC for admin? Also, upon login, the super admin team said they want the redirect URL to be /super-admin/analytics-and-reporting instead of the one they suggested earlier.

Reply:
> Product listing is not the only admin route, please can you wrap the other admin routes with the HOC for admin? Also, upon login, the super admin team said they want the redirect URL to be /super-admin/analytics-and-reporting instead of the one they suggested earlier. 

Alright, I will do that.